frontend/src/screens/achievements.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Text, Button, Container, Title, Tabs, RingProgress, Image, Group, Stack, Flex, Modal, Overlay, Alert } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import { useApi } from '@lib';
+import { launchStudy } from '@models';
+import { TopNavBar, Footer } from '@components';
+import { colors } from '@theme';
+import { useFetchLearningPaths } from './learner/studies';
+import { ParticipantStudy } from '@api';
+import { useCurrentUser } from '@lib';
+
+const AchievementBadge:FC<{learningPath: string; studies: ParticipantStudy[]}> = ({ learningPath, studies }) => {
+
+    interface Criteria{
+        title: string,
+        text: string
+    }
+
+    const api = useApi()
+    const user = useCurrentUser()
+    const [progress, setProgress] = useState(0)
+    const [isComplete, setIsComplete] = useState(false)
+    const [criteria, setCriteria] = useState<Criteria[]>([] || undefined)
+    const [completedStudies, setCompletedStudies] = useState<ParticipantStudy[]>([])
+    const [btnText, setBtnText] = useState('Start')
+    const [error, setError] = useState(false)
+    const [opened, { open, close }] = useDisclosure(false)
+
+    useEffect(() => {
+        setCompletedStudies(studies.filter((study: ParticipantStudy) => study.completedAt))
+        setCriteria(() => {
```

Comment:
Criteria is just HTML right? I forget since its been so long (is it markdown? or html?) if so - later on we can probably just use [dangerouslySetInnerHTML](https://blog.logrocket.com/using-dangerouslysetinnerhtml-react-application/) since the HTML is coming from our badges and its not set by anyone else

Reply:
It is markdown